packages/ckeditor5-image/src/imageutils.ts

```diff
@@ -341,7 +342,7 @@ function isNotInsideImage( selection: DocumentSelection ): boolean {
  * Returns a node that will be used to insert image with `model.insertContent`.
  */
 function getInsertImageParent( selection: Selection | DocumentSelection, model: Model ): Element | DocumentFragment {
-	const insertionRange = findOptimalInsertionRange( selection, model );
+	const insertionRange = model.schema.findOptimalInsertionRange( selection, model );
```

Comment:
This was using the widget implementation of this method. I think this should not change.

Reply:
Adressed in 24ac1fa